src/client.ts

```diff
@@ -241,6 +243,23 @@ export class PluggyClient extends BaseApi {
     return this.createGetRequest(`investments/${id}`, null, transformInvestment)
   }
 
+  /**
+   * Fetch transactions from an investment
+   * @param investmentId The investment id
+   * @param {TransactionFilters} options Transaction options to filter
+   * @returns {PageResponse<InvestmentTransaction[]>} object which contains the transactions list and related paging data
+   */
+  async fetchInvestmentTransactions(
+    investmentId: string,
+    options: TransactionFilters = {}
+  ): Promise<PageResponse<InvestmentTransaction>> {
+    return this.createGetRequest(
+      `investments/${investmentId}/transactions`,
+      { ...options, investmentId },
```

Comment:
There is no need to pass `investmentId` as a query parameter here, right? We are using it in the endpoint.
```suggestion
      { ...options },
```
Or I'm missing something 😄

Reply:
I agree, I just left it the same as `transactions` endpoint to be consistent. Maybe another day we can refactor this and remove it from anywhere